packages/grafana-ui/src/components/MatchersUI/utils.ts

```diff
@@ -79,9 +79,20 @@ export function useSelectOptions(
       if (!found && name === currentName) {
         found = true;
       }
+      const field = displayNames.fields.get(name);
+      let previewValues = '';
+      if (field?.values.length && field.values.length > 2) {
+        previewValues += field.values.get(0);
```

Comment:
rather than the raw value, should we use the formatted one?  ie something like:
```
const disp = (v:any) => formattedValueToString( field.diplay!(v) );
previewValues += disp(field.values.get(0));
...
```